pkg/fwdservice/fwdservice.go

```diff
@@ -76,7 +79,8 @@ type ServiceFWD struct {
 	ForwardIPReservations    []string // cli passed IP reservations
 }
 
-/**
+/*
+*
```

Comment:
```suggestion
/**
```
I wouldn't make this change, it's not relevant to the rest of the PR.